src/plugins/coreplugin/dialogs/externaltoolconfig.cpp

```diff
@@ -619,7 +619,7 @@ void ExternalToolConfig::updateEffectiveArguments()
 
 void ExternalToolConfig::editEnvironmentChanges()
 {
-    const QString placeholderText = Utils::HostOsInfo::isWindowsHost()
+    const QString placeholderText = Utils::HostOsInfo::isWindowsHost() || Utils::HostOsInfo::isOs2Host()
```

Comment:
```suggestion
    const QString placeholderText = Utils::HostOsInfo::isDosLikeHost()
```
See previous commit.